app/controllers/questions_controller.rb

```diff
@@ -30,6 +30,17 @@ def show
     @answer = @question.answers.new
   end
 
+  def destroy
+    @question = Question.find(params[:id])
+    if @question.delete
+      flash[:success] = "Your question was deleted successfully!"
+      redirect_to root_path
+    else
+      flash[:danger] = "Failed to delete the question"
```

Comment:
I believe *redirect_to request.referrer* can get you back to the page you came from. You might play with this. I believe request refer gives you the url of where the request came from so that you can redirect back. This seems logical if I'm on an index page with multiple questions but not on a show. Tough call. Anyone else have feedback on this? @andrewdwooten @Ryanspink1 @TheKaterTot @maxglassie